### DISCUSSION TITLE
Can't login with local accaunt

### DISCUSSION THREAD

**Discussion by fropa** [apache/superset#17189] [Q&A / Help]
Hi, I integrated superset with the LDAP, and finally, it works. But now, I cannot log in with a local (admin) account, if I comment LDAP integration configuration in config.py then I can log in but with LDAP I can't.

### Expected results

I expected to login with both, LDAP and local accaunts.

### Actual results

I have to choose one of them.

### Environment

(please complete the following information):

- superset version: `0.38.1`
- python version: `3.6.8`
- node.js version: `node -v`

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

Also, I want to configure multiple integrations with the same LDAP, for x group 1 role, for y group 2 role, etc... Which part is responsible for that??

**Comment by srinify** [comment]
Hi @fropa to be honest, this is probably a better question for Stack Overflow. You can use the apache-superset tag: https://stackoverflow.com/questions/tagged/apache-superset

Can you provide significantly more context on your setup so people here on Github or on StackOverflow can *try* to help?

**Comment by fropa** [comment]
> Hi @fropa to be honest, this is probably a better question for Stack Overflow. You can use the apache-superset tag: https://stackoverflow.com/questions/tagged/apache-superset
> 
> Can you provide significantly more context on your setup so people here on Github or on StackOverflow can _try_ to help?

Hi srinify, thank you for your answer, I'll ask on StackOverflow and provide more details here too.

I did that context : 


```
from flask_appbuilder.security.manager import AUTH_OID
from flask_appbuilder.security.manager import AUTH_REMOTE_USER
from flask_appbuilder.security.manager import AUTH_DB
from flask_appbuilder.security.manager import AUTH_LDAP
```

```
AUTH_TYPE = AUTH_LDAP
AUTH_USER_REGISTRATION = True
AUTH_USER_REGISTRATION_ROLE = "Public"

AUTH_LDAP_SERVER = "ldap://10.10.0.50"
AUTH_LDAP_USE_TLS = False
AUTH_LDAP_BIND_USER = "CN=Administrator,CN=Users,DC=gru,DC=lab"
AUTH_LDAP_BIND_PASSWORD = "password"
AUTH_LDAP_SEARCH = "DC=gru,DC=lab"
AUTH_LDAP_GROUP_FIELS="CN=linuxadmins,DC=gru,DC=lab"
AUTH_LDAP_SEARCH = "DC=gru,DC=lab"
AUTH_LDAP_UID_FIELD = "sAMAccountName"
AUTH_LDAP_FIRSTNAME_FIELD = "givenName"
AUTH_LDAP_LASTNAME_FIELD = "sn"
```
with that configuration I can log in to superset web, (I'm trying to split login roles with "AUTH_ROLES_MAPPING" but with no luck yet) but when I log in through LDAP users, I can't use local user such as admin (the user which created during installation).

**Comment by philipf-chen** [comment]
> > Hi @fropa to be honest, this is probably a better question for Stack Overflow. You can use the apache-superset tag: https://stackoverflow.com/questions/tagged/apache-superset
> > Can you provide significantly more context on your setup so people here on Github or on StackOverflow can _try_ to help?
> 
> Hi srinify, thank you for your answer, I'll ask on StackOverflow and provide more details here too.
> 
> I did that context :
> 
> ```
> from flask_appbuilder.security.manager import AUTH_OID
> from flask_appbuilder.security.manager import AUTH_REMOTE_USER
> from flask_appbuilder.security.manager import AUTH_DB
> from flask_appbuilder.security.manager import AUTH_LDAP
> ```
> 
> ```
> AUTH_TYPE = AUTH_LDAP
> AUTH_USER_REGISTRATION = True
> AUTH_USER_REGISTRATION_ROLE = "Public"
> 
> AUTH_LDAP_SERVER = "ldap://10.10.0.50"
> AUTH_LDAP_USE_TLS = False
> AUTH_LDAP_BIND_USER = "CN=Administrator,CN=Users,DC=gru,DC=lab"
> AUTH_LDAP_BIND_PASSWORD = "password"
> AUTH_LDAP_SEARCH = "DC=gru,DC=lab"
> AUTH_LDAP_GROUP_FIELS="CN=linuxadmins,DC=gru,DC=lab"
> AUTH_LDAP_SEARCH = "DC=gru,DC=lab"
> AUTH_LDAP_UID_FIELD = "sAMAccountName"
> AUTH_LDAP_FIRSTNAME_FIELD = "givenName"
> AUTH_LDAP_LASTNAME_FIELD = "sn"
> ```
> 
> with that configuration I can log in to superset web, (I'm trying to split login roles with "AUTH_ROLES_MAPPING" but with no luck yet) but when I log in through LDAP users, I can't use local user such as admin (the user which created during installation).

@fropa Same issue, any solutions?
